Tests/LibWeb/Text/input/Crypto/SubtleCrypto-sign.html

```diff
@@ -0,0 +1,32 @@
+<script src="../include.js"></script>
+<script>
+    asyncTest(async done => {
+        const encoder = new TextEncoder();
+        const message = "Hello friends";
+        const encoded_message = encoder.encode(message);
+
+        const key_algorithm = {
```

Comment:
I believe all JavaScript should be using camelCase. We should figure out how to get prettier to check our html test files :thinking: (@trflynn89 thoughts?)

Reply:
Yeah I run prettier on test files I add and it works well enough. Linting in CI sounds good, but I don't think it handles snake vs. camel case.

Reply:
Swapped to camelCase 👍